pkg/deployment/deployment.go

```diff
@@ -607,3 +607,17 @@ func (builder *Builder) WithToleration(toleration coreV1.Toleration) *Builder {
 
 	return builder
 }
+
+// WithSchedulerName configures a scheduler to process pod's scheduling.
+func (builder *Builder) WithSchedulerName(schedulerName string) *Builder {
```

Comment:
Please add validation that schedulerName is not empty string

Reply:
thanks, updated